docs/reference-guides/block-api/block-bindings.md

```diff
@@ -0,0 +1,296 @@
+# Bindings
+
+Block Bindings API lets you “bind” dynamic data to the block’s attributes, which are then reflected in the final HTML markup that is output to the browser on the front end.
+
+An example could be connecting an Image block `url` attribute to a function that returns random images from an external API.
+
+```html
+<!-- wp:image {
+	"metadata":{
+		"bindings":{
+			"url":{
+				"source":"my-plugin/get-random-images"
+			}
+		}
+	}
+} -->
+```
+
+
+## Compatible blocks and its attributes
+
+Right now, not all blocks attributes are compatible with block bindings. There is some effort going on increasing this compatibility, but for now, this is the list:
```

Comment:
"blocks attributes" should be "block attributes"

Reply:
Done in https://github.com/WordPress/gutenberg/pull/66251/commits/b553fcd81feba6985e24825ec252bc0aee2585fc